src/Helper/DbArrayHelper.php

```diff
@@ -289,16 +289,16 @@ public static function toArray(array|object $object): array
         return get_object_vars($object);
     }
 
-    public static function queryAsArray(array|ExpressionInterface|string $query): array
+    public static function normalizeToArray(array|ExpressionInterface|string $expression): array
```

Comment:
```suggestion
    /**
     * Normalizes raw input into an array of expression values.
     *
     * @param array|ExpressionInterface|string $raw Raw input to be normalized. It can be:
     *  - an array of expression values;
     *  - a single expression object;
     *  - a string with comma-separated expression values.
     *
     * @return array An array of normalized expressions.
     */
    public static function normalizeExpressions(array|ExpressionInterface|string $raw): array
```